src/classes/TestInvoiceController.cls

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by ociszek001 on 28.05.2019.
+ */
+
+@IsTest
+private class TestInvoiceController {
+    @TestSetup
+    static void loadInvoices() {
+        User u = TestDataClass.insertUser();
+        List<Invoice__c> inv = new List<Invoice__c>();
+        System.runAs(u) {
+            Test.startTest();
+            VAT__c vat = TestDataClass.insertVat();
+            inv = TestDataClass.loadInvoices();
+            List<Service__c> ser = TestDataClass.insertServices(new List<VAT__c>{
+                    vat
+            }, inv);
+            Test.stopTest();
+        }
+        System.assertEquals(false, inv.isEmpty());
+    }
+
+    @isTest
+    static void testGetInvoice() {
+        User u = [SELECT Id from User where Username = :TestDataClass.userName];
+        Invoice__c invoice = new Invoice__c();
+        System.runAs(u) {
+            Test.startTest();
+            List<Invoice__c> inv = [SELECT Id from Invoice__c LIMIT 1];
+            PageReference invoicePage = Page.Invoice;
+            Test.setCurrentPage(invoicePage);
+            ApexPages.currentPage().getParameters().put('invId', inv.get(0).Id);
+            InvoiceController invC = new InvoiceController();
+            invoice = invC.getInvoice();
+            Test.stopTest();
+        }
+        System.assertEquals('Unpaid', invoice.Status__c);
+    }
+
+    @isTest
+    static void testGetLogos() {
+        User u = [SELECT Id from User where Username = :TestDataClass.userName];
+        ContentDocumentLink cdl = new ContentDocumentLink();
+        Test.startTest();
+        Account acc = new Account(Name = 'TestTest4', NIP__c = '115-111-11-11', Phone = '113', BillingPostalCode = '12-542');
+        insert acc;
+        ContentVersion contentVersion = new ContentVersion(
```

Comment:
Insertujesz ContentVersion w kilku miejsach. Może warto przenieść to do naszej TestDataClasy i tutaj jej uzywać ?

Reply:
Insertuje contentVersion w dwóch miejscach:
- w teście samej klasy - i tam potrzebuje zainsertować dwie rożne ContentVersion
- w teście InvoiceController
To za dużo?

Reply:
Może się przydać na późniejsze testy. W 1 przypadku ok zostaw tak jak jest, w 2 przypadku zrób metodę w Test Data Klas i jej użyj.

Reply:
a normalnie zrobię zapytanie o Id do Content Version?